lesson07/assignment/assignment07/src/_components.content.scss

```diff
@@ -0,0 +1,48 @@
+* {
+    font-family: "Georgia", serif;;
+}
+
+
+.navbar {
+background-color: #f2e6df;
+border-radius: 25px;
+}
+
+//car icon
+i[class^="fas"] {
+  color: #de1616;
```

Comment:
If this is a color you anticipate using more places in the design, it should be made into a variable. If this is a one-off, it's fine to leave it here as just a hex.